samples/bot-configuration-app/nodejs/Readme.md

```diff
@@ -0,0 +1,133 @@
+---
+page_type: sample
+description: Demonstrating the feature of configurable card with type ahead search (static and dynamic) control on Adaptive Cards.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
```

Comment:
All screeshots are wrong because of footer and .gif as well
![Uploading image.png…]()

Reply:
Resolved